app/controllers/api/v1/users_controller.rb

```diff
@@ -98,4 +101,9 @@ def revoke_doorkeeper_request_token!
     token = Doorkeeper.authenticate(request)
     token.revoke
   end
+
+  def user_info_changed(user_id, changed)
+    # Mailers::UserInfoChanged.run!(user_id, changed)
+    operation.run!(user_id, changed)
```

Comment:
I'd just do it like you commented out in this case. The `operation` helper is more for when we're moving the entire action body into the operation (in my mind at least)